envi/archs/arm/disasm.py

```diff
@@ -2562,9 +2588,9 @@ def repr(self, op):
             addr = self.getOperAddr(op)    # only works without an emulator because we've already verified base_reg is PC
             tname = "[#0x%x]" % addr
             # FIXME: is there any chance of us doing indexing on PC?!?
-            # ldcl literal trips this in some cases - leaving for now - to check for other instances
-            if idxing != 0x10:
-                print "OMJ! indexing on the program counter!"
+            # ldcl literal trips this in some cases
+            #if idxing != 0x10:
+            #    print "OMJ! indexing on the program counter!"
```

Comment:
i need to investigate these and probably just remove them.  we seem to index on PC more often than we originally thought possible.